src/content/en/2021/compression.md

```diff
@@ -169,11 +157,11 @@ From these results we can see that, [*compared to 2020*](../2020/compression#fir
 
 ## Compression levels
 
-Compression level is a parameter given to the encoder to adjust the amount of effort is applied to find redundancy in the input in order to consequently achieve higher compression density.
+Compression level is a parameter given to the encoder to adjust the amount of effort is applied to find redundancy in the input in order to consequently achieve higher compression density. A higher compression level results in slower compression, but does not substantially affect the decompression speed, even making it slightly faster. For precompressed content, the time to compress the data has no effect on the user experience because it was already compressed beforehand. For dynamic content, the amount time the CPU needs to compress can be traded off to the gain in speed to send the data over the network.
```

Comment:
```suggestion
Compression level is a parameter given to the encoder to adjust the amount of effort is applied to find redundancy in the input in order to consequently achieve higher compression density. A higher compression level results in slower compression, but does not substantially affect the decompression speed, even making it slightly faster. For precompressed content, the time needed to compress the data has no effect on the user experience because it can be done beforehand. For dynamic content, the amount time the CPU needs to compress the resource can be traded off to the gain in speed to send the reduced, compressed data over the network.
```